core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -371,7 +371,7 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
     def googleProjectExists(projectId: GoogleProjectId) =
```

Comment:
Would it be worth renaming this function? Since it now returns false for projects that do exist but can't be accessed? Though since it looks like it's local to `deleteGoogleProjectIfChild`, the potential for misunderstanding is probably low.

Reply:
Good call, renamed it. Let me know if I made it worse 😂